packages/swirl-components/src/components/swirl-form-control/swirl-form-control.tsx

```diff
@@ -44,6 +44,7 @@ export class SwirlFormControl {
   @Prop() label!: string;
   @Prop() labelPosition?: SwirlFormControlLabelPosition = "inside";
   @Prop() tooltip?: string;
+  @Prop() secondaryLabel?: string;
```

Comment:
**question:** **line 129  `setInputElementLabel()` :** I was wondering if the label of the actual input element should now consists of both properties `label` and `secondaryLabel`? What do you think?

Reply:
good question. probably highly dependend on the value the secondary label gives., but we could just assume that yes. so i will implement it :)